notebooks/241-freevc/241-freevc.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>please align model parts names with mentioned before, it is difficult to find correlation between diagram and names in this paragraph.</p><p>Also probably title should be Convert Modes to OpenVINO Intermediate Representation instead of Use Model Optimizer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>